packages/core/src/utils/assert-sign-in-experience.ts

```diff
@@ -0,0 +1,26 @@
+import { Branding, BrandingStyle, TermsOfUse } from '@logto/schemas';
+
+import assertThat from '@/utils/assert-that';
+
+export const assertBranding = (branding?: Branding) => {
+  if (!branding) {
+    return;
+  }
+
+  if (branding.style === BrandingStyle.Logo_Slogan) {
+    assertThat(
+      branding.slogan && branding.slogan.trim() === '',
+      'sign_in_experiences.empty_slogan'
+    );
+  }
+};
+
+export const assertTermsOfUse = (termsOfUse?: TermsOfUse) => {
+  if (!termsOfUse) {
+    return;
+  }
+
+  if (termsOfUse.enabled) {
+    assertThat(termsOfUse.contentUrl, 'sign_in_experiences.empty_content_url_of_terms_of_use');
+  }
+};
```

Comment:
same.

Reply:
Write in a single assertThat like this:
```ts
export const assertTermsOfUse = (termsOfUse?: TermsOfUse) => {
  assertThat(
    !termsOfUse || !termsOfUse.enabled || termsOfUse.contentUrl,
    'sign_in_experiences.empty_content_url_of_terms_of_use'
  );
};
```
Since it's less readable, current implementation is better.

Reply:
```ts
assertThat(!termsOfUser?.enabled || termsOfUser.contentUrl, 'xxx');
```